query-connector/src/app/queryBuilding/buildFromTemplates/page.tsx

```diff
@@ -95,100 +115,85 @@ export default function QueryTemplateSelection() {
     });
   };
 
-  const submitForm = () => {
-    // TODO: do something with selectedConditions on next step/page
-    // will be addressed in https://linear.app/skylight-cdc/issue/QUE-65/create-the-valueset-selection-page
-    console.log(selectedConditions);
-  };
-
   const atLeastOneItemSelected =
     selectedConditions && Object.values(selectedConditions).length > 0;
 
   return (
     <>
       <SiteAlert />
-      <div className="main-container__wide">
+      <div
+        className="main-container__wide"
+        style={{
```

Comment:
could we pull these inline styles into either the css module file or use [the inline utility classes?](https://designsystem.digital.gov/utilities/flex/)